rust/src/logger.rs

```diff
@@ -0,0 +1,39 @@
+//! Logging module
```

Comment:
Right, gotcha. So would this be transparent in the sense that it would still show the correct file and line number in Suricata's output?

Function name seems to be tricky in Rust.